src/utils/normalizePath.js

```diff
@@ -1,11 +1,27 @@
+const memo = new Map()
```

Comment:
Is it really important? This will grow and use memory. Maybe using WeakMap here will be a better option. What do you think?

Reply:
I'm not sure WeakMap would be appropriate as it needs object, but I see your point.

If memory consideration is important then I think the improvement without the memoization is enough.

If we can use memory then just memoizing this method improve performances further (+10%). The cost of memoizing this method memory wize is not that huge because the map contains only strings.

Your call, I don't mind removing it :)

Reply:
Let's keep it and if it will consume too much memory we will remove it later.